AutoRest/Generators/Extensions/Extensions.Tests/MappingTests.cs

```diff
@@ -33,10 +33,9 @@ public void TestInputMapping()
             generator.Generate(clientModel).GetAwaiter().GetResult();
             string body = settings.FileSystem.ReadFileAsText("X:\\Payload.cs");
```

Comment:
I talked to Dan and he disabled it because it was failing due to the code gen change